### PR TITLE
check for makemessages failures and warn

### PR DESCRIPTION
**Pull request by esoergel**
@dannyroberts
I think this has been failing for about a month, but the script still completed, so we just went with it

### CODE REVIEW

File: scripts/update-translations.sh [hunk 1]
```diff
@@ -33,9 +33,19 @@ echo "Pulling translations from transifex"
 tx pull -f
 
 echo "Gathering all translation strings.  Note that this will probably take a while"
-./manage.py makemessages --all
+./manage.py makemessages --all --ignore 'corehq/apps/app_manager/tests/data/v2_diffs*' --ignore 'node_modules'
```
Comment (esoergel) [inline scripts/update-translations.sh]: These diffs have bad stuff in them, figure it's best just to skip.

Comment (dannyroberts) [inline scripts/update-translations.sh]: Those are definitely both good to skip, we wouldn't want to pull translation-looking things from there even if it didn't happen to crash, so this is an improvement all around

**Review by dannyroberts — COMMENTED** [review]
Cool, looks like a nasty problem and a great solution!

File: scripts/update-translations.sh (hunk 1 above)
Comment (benrudolph) [inline scripts/update-translations.sh]: 👍

**Review by benrudolph — APPROVED** [review]

**Pull request merged**